tests/ops/qubit/test_parametric_ops.py

```diff
@@ -2768,10 +2768,16 @@ def test_PauliRot_all_Identity(self):
         op = qml.PauliRot(theta, "II", wires=[0, 1])
         decomp_ops = op.decomposition()
 
-        assert np.allclose(op.eigvals(), np.exp(-1j * theta / 2) * np.ones(4))
-        assert np.allclose(op.matrix() / op.matrix()[0, 0], np.eye(4))
+        assert len(decomp_ops) == 1
+
+        decomp_op = decomp_ops[0]
 
-        assert len(decomp_ops) == 0
+        assert decomp_op.name == "GlobalPhase"
+
+        # global phase acts on all wires so wire attribute is unused
+        assert decomp_op.wires == Wires([])
+        assert qml.math.allclose(decomp_op.data[0], theta / 2)
+        assert qml.math.allclose(op.matrix(), decomp_op.matrix() * np.eye(4))
```

Comment:
```suggestion
        assert qml.equal(decomp_op, qml.GlobalPhase(theta/2))
```

Reply:
I totally forgot about qml.equal for some reason 😅. Thank you!

Reply:
Actually, don't we want to check also that the decomposition is equal to the original operator? Hence what I had before? Currently we are only checking the the decomposition is equal to the Global Phase

Reply:
So I would add this back:
```
assert qml.math.allclose(op.matrix(), decomp_op.matrix() * np.eye(4))
```

Reply:
@albi3ro is this resolved?